demos/python_demos/handwritten_japanese_recognition_demo/README.md

```diff
@@ -50,13 +52,27 @@ Options:
                         Optional. Number of inference iterations
   -cl CHARLIST, --charlist CHARLIST
                         Path to the decoding char list file
+  -dc DESIGNATED_CHARACTERS, --designated_characters DESIGNATED_CHARACTERS
+                        Optional. Path to the designated character file
+  -tk TOP_K, --top_k TOP_K
+                        Optional. Top k steps in looking up the decoded
+                        character, until a designated one is found
 ```
 
 
 For example:
 ```
 python handwritten_japanese_recognition_demo.py -i data/test.png -m path/to/ir_xml/model.xml
 
+```
+
+When the `designated_characters` argument is provided, the script will check Top k steps in looking up the decoded character, until a designated one is found. By doing so, the output character will be restricted to a designated region. K is set to 20 by default.
+
+For example, if we want to restrict the output character to a digit and hyphen circumstance, we need to provide the path to the designated character file, then the command line will be like:
```

Comment:
```suggestion
For example, if we want to restrict the output character only to digits and hyphen circumstance, we need to provide the path to the designated character file and set K to the number of rows in charlist file, then the command line will be like:
```
The initial version misled me. I expected that only the chars from `digit_hyphen.txt` will appear, but other characters are still allowed if none of `digit_hyphen.txt` is in first K chosen elements.

Reply:
Actually, your initial expectation is right. This pull request is based on the charlist file.  But if an additional designated_characters file is provided, this script will perform a **post**-filtering processing, which means it is possible that other characters are still allowed if none of digit_hyphen.txt is in first K chosen elements. Thus, the -tk argument has little to do with the charlist file, instead it is an extra argument for post-processing.

Reply:
I think I was unclear with my comment. I meant that

I expected that only the chars from `digit_hyphen.txt` will appear, but *the reality turned out to be that* other characters are still allowed if none of `digit_hyphen.txt` is in first K chosen elements.

> post-filtering processing, which means it is possible that other characters are still allowed if none of digit_hyphen.txt is in first K chosen elements.

So that's what I was trying to point out, but your version is clearer :) Could you add it to readme?

Reply:
Sure. Thanks your careful review